src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Xproj/MigrateXprojProjectFactory.cs

```diff
@@ -197,6 +210,23 @@ internal bool BackupProject(string backupLocation, string xprojLocation, string
             return (report.OutputMSBuildProject, report.Succeeded);
         }
 
+        internal void CleanupXproj(string projectLocation, string projectName)
+        {
+            // Clean up the .xproj, the project.json, the project.lock.json, and the xproj.user
+            _fileSystem.RemoveFile(Path.Combine(projectLocation, $"{projectName}.xproj"));
```

Comment:
Because the others have to exist: we've already proved that. The lock file and .user files don't have to exist, so if an IOException is thrown while removing them we don't care.